### PR TITLE
executor: Fix show create table with multiple fk

### PR DESCRIPTION
**Pull request by shenli**
Use comma to separate multiple foreign keys.

### CODE REVIEW

File: executor/show.go [hunk 1]
```diff
@@ -451,6 +455,7 @@ func (e *ShowExec) fetchShowCreateTable() error {
 		if ast.ReferOptionType(fk.OnUpdate) != ast.ReferOptionNoOption {
 			buf.WriteString(fmt.Sprintf(" ON UPDATE %s", ast.ReferOptionType(fk.OnUpdate)))
 		}
+
```
Comment (hanfei1991) [inline executor/show.go]: remove blank ?

File: executor/show_test.go [hunk 2]
```diff
@@ -125,6 +126,25 @@ func (s *testSuite) TestShow(c *C) {
...
 	result = tk.MustQuery("SHOW index from show_index from test where Column_name = 'c'")
 	c.Check(result.Rows(), HasLen, 1)
+
+	// For table with multiple fk.
+	testSQL = `CREATE TABLE pilot_languages (
+			pilot_id int(11) NOT NULL,
+			language_id int(11) NOT NULL,
+			CONSTRAINT pilot_language_fkey FOREIGN KEY (pilot_id) REFERENCES pilots (pilot_id),
+		        CONSTRAINT languages_fkey FOREIGN KEY (language_id) REFERENCES languages (language_id)
+		) ENGINE=InnoDB;`
```
Comment (coocood) [inline executor/show_test.go]: You can define the string like this, then use `testSQL` in expected row value.
```
    testSQL = `CREATE TABLE pilot_languages (
  pilot_id int(11) NOT NULL,
  language_id int(11) NOT NULL,
  CONSTRAINT pilot_language_fkey FOREIGN KEY (pilot_id) REFERENCES pilots (pilot_id),
  CONSTRAINT languages_fkey FOREIGN KEY (language_id) REFERENCES languages (language_id)
) ENGINE=InnoDB`
```

Comment (coocood) [inline executor/show_test.go]: Never mind, the result contains backquote.

Comment (shenli) [inline executor/show_test.go]: Yes, you get it.

**Comment by shenli** [comment]
@coocood @hanfei1991 PTAL

**Comment by coocood** [comment]
LGTM

**Pull request merged**